README.md

```diff
@@ -306,6 +306,28 @@ Generated shape:
 <http://www.ldbc.eu/ldbc_socialnet/1.0/vocabulary/Person> a rdfs:Class.
 ```
 
+#### Vocabuary Predicate Domain Handler
+
+Generates vocabulary information.
```

Comment:
```suggestion
Generates vocabulary information about the domain of a specific predicate.
```

Reply:
It didn't get pushed earlier have now made the changes